Compute/src/V1/Gapic/AcceleratorTypesGapicClient.php

```diff
@@ -225,76 +227,77 @@ public function __construct(array $options = [])
+     *           If no page token is specified (the default), the first page
+     *           of values will be returned. Any page token used here must have
+     *           been generated by a previous call to the API.
      *     @type bool $returnPartialSuccess
-     *          Opt-in for partial success behavior which provides partial results in case of failure. The default value is false and the logic is the same as today.
+     *           Opt-in for partial success behavior which provides partial results in case of failure. The default value is false and the logic is the same as today.
      *     @type RetrySettings|array $retrySettings
-     *          Retry settings to use for this call. Can be a
-     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
-     *          of retry settings parameters. See the documentation on
-     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     *           Retry settings to use for this call. Can be a
+     *           {@see Google\ApiCore\RetrySettings} object, or an associative array of retry
+     *           settings parameters. See the documentation on
+     *           {@see Google\ApiCore\RetrySettings} for example usage.
      * }
      *
      * @return \Google\ApiCore\PagedListResponse
      *
      * @throws ApiException if the remote call fails
-     * @experimental
      */
     public function aggregatedList($project, array $optionalArgs = [])
     {
         $request = new AggregatedListAcceleratorTypesRequest();
+        $requestParamHeaders = [];
         $request->setProject($project);
+        $requestParamHeaders['project'] = $project;
         if (isset($optionalArgs['filter'])) {
             $request->setFilter($optionalArgs['filter']);
         }
+
         if (isset($optionalArgs['includeAllScopes'])) {
             $request->setIncludeAllScopes($optionalArgs['includeAllScopes']);
         }
+
         if (isset($optionalArgs['maxResults'])) {
             $request->setMaxResults($optionalArgs['maxResults']);
         }
+
         if (isset($optionalArgs['orderBy'])) {
             $request->setOrderBy($optionalArgs['orderBy']);
         }
+
         if (isset($optionalArgs['pageToken'])) {
             $request->setPageToken($optionalArgs['pageToken']);
         }
+
         if (isset($optionalArgs['returnPartialSuccess'])) {
             $request->setReturnPartialSuccess($optionalArgs['returnPartialSuccess']);
         }
 
-        return $this->getPagedListResponse(
-            'AggregatedList',
-            $optionalArgs,
-            AcceleratorTypeAggregatedList::class,
-            $request
-        );
+        $requestParams = new RequestParamsHeaderDescriptor($requestParamHeaders);
+        $optionalArgs['headers'] = isset($optionalArgs['headers']) ? array_merge($requestParams->getHeader(), $optionalArgs['headers']) : $requestParams->getHeader();
```

Comment:
Can you tell me a bit what this is for? I am unfamiliar with this change

Reply:
Resolved offline, please see [this PR](https://github.com/googleapis/gapic-generator-php/pull/234).